### PR TITLE
FOGL-2815: adapt FogLAMP readme files to reference RedHat/Centos

### PR DESCRIPTION
**Pull request by sstefano69**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -69,6 +69,20 @@ FogLAMP is currently based on C/C++ and Python code. The packages needed to buil
 - sqlite3
 - libsqlite3-dev
 
+
+Linux distributions
+-------------------
+
+FogLAMP can be build or installed in one of the following Linux distributions :
```
Comment (MarkRiddoch) [inline README.rst]: built not build

File: README.rst [hunk 2]
```diff
@@ -69,6 +69,20 @@ FogLAMP is currently based on C/C++ and Python code. The packages needed to buil
...
+
+- Ubuntu
```
Comment (MarkRiddoch) [inline README.rst]: Best to call out Ubuntu 16 and Ubuntu 18 explicitly

File: README.rst [hunk 3]
```diff
@@ -69,6 +69,20 @@ FogLAMP is currently based on C/C++ and Python code. The packages needed to buil
...
+- Raspbian
+- Red Hat
+- CentOS
```
Comment (MarkRiddoch) [inline README.rst]: Also Red Hat 7.6 and SentOS 7.6 - we should be clear we are not including CentOS or RHEL 8

File: README.rst (hunk 1 above)
Comment (sstefano69) [reply README.rst]: done

File: README.rst (hunk 2 above)
Comment (sstefano69) [reply README.rst]: done

File: README.rst (hunk 3 above)
Comment (sstefano69) [reply README.rst]: done

File: REDHAT.rst [hunk 4]
```diff
@@ -0,0 +1,45 @@
+.. |br| raw:: html
+
+   <br />
+
+.. Links
+.. _download page: https://foglamp.readthedocs.io/en/master/92_downloads.html
```
Comment (Singhal-Vaibhav) [inline REDHAT.rst]: This has been changed to http://dianomic.com/download-packages/ Refer docs/92_downloads.rst

File: REDHAT.rst [hunk 5]
```diff
@@ -0,0 +1,45 @@
+.. |br| raw:: html
```
Comment (Singhal-Vaibhav) [inline REDHAT.rst]: Not sure if root is the correct place to place this rst file, I can see RESPBIAN.rst file there too which should also be removed. All the .rst files should be placed inside docs and their entry point should be defined in docs/index.rst. This is how docs will be compile and build for readthedocs.

File: REDHAT.rst [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+.. _foglamp-pkg: https://github.com/foglamp/foglamp-pkg
+
+*******************************************
+Building and using FogLAMP on RedHat/CentOS
+*******************************************
+
+FogLAMP can be build or installed on Red Hat or CentOS, it is currently tested against:
+
+- Red Hat 7.6
+- CentOS  7.6-1810
```
Comment (Singhal-Vaibhav) [inline REDHAT.rst]: Please remove the patch number -1810

Comment (sstefano69) [reply REDHAT.rst]: done

File: REDHAT.rst (hunk 4 above)
Comment (sstefano69) [reply REDHAT.rst]: done

File: README.rst [hunk 7]
```diff
@@ -80,6 +94,14 @@ On Ubuntu-based Linux distributions the packages can be installed with given `re
 
 You may need to use *sudo* to allow *apt-get* to install packages dependent upon your access rights.
 
+Install the prerequisites on Red Hat/CentOS
+-------------------------------------------
+
+On Red Hat and CentOS distributions the required packages can be installed automatically with given `requirements.sh <requirements.sh>`_:
+::
+	sh ./requirements.sh
```
[the remark this answers is not in the document]
Comment (sstefano69) [reply README.rst]: done

File: REDHAT.rst (hunk 5 above)
Comment (sstefano69) [reply REDHAT.rst]: done as suggested.

File: README.rst [hunk 8]
```diff
@@ -79,6 +93,14 @@ On Ubuntu-based Linux distributions the packages can be installed with given `re
...
 You may need to use *sudo* to allow *apt-get* to install packages dependent upon your access rights.
 
+Install the prerequisites on Red Hat/CentOS
+-------------------------------------------
+
+On Red Hat and CentOS distributions the required packages can be installed automatically with given `requirements.sh <requirements.sh>`_:
+::
+	./requirements.sh
+
+You may need to use *sudo* upon your access rights.
```
Comment (MarkRiddoch) [inline README.rst]: You should run this as a user with *sudo* access rights.

File: docs/REDHAT.rst [hunk 9]
```diff
@@ -0,0 +1,45 @@
...
+
+.. Links
+.. _download page: http://dianomic.com/download-packages
+.. _foglamp-pkg: https://github.com/foglamp/foglamp-pkg
+
+*******************************************
+Building and using FogLAMP on RedHat/CentOS
+*******************************************
+
+FogLAMP can be build or installed on Red Hat or CentOS, it is currently tested against:
```
Comment (MarkRiddoch) [inline docs/REDHAT.rst]: "built" not "build"

File: docs/REDHAT.rst [hunk 10]
```diff
@@ -0,0 +1,45 @@
...
+
+The FogLAMP RPM is available in the download page of the documentation available at `download page`_.
+
+The RPM can also be created from the FogLAMP sources through the repository `foglamp-pkg`_ using the make_rpm script and following the instruction in the README.rst.
+
+
+Installation on Red Hat
+=======================
+
+A Red Hat package should be installed before the FogLAMP RPM, follow the instructions :
```
Comment (MarkRiddoch) [inline docs/REDHAT.rst]: It is necessary to install a Red Hat package before FogLAMP can be installed successfully. The installation sequence is as follows:

File: docs/REDHAT.rst [hunk 11]
```diff
@@ -0,0 +1,45 @@
...
+::
+   sudo yum-config-manager --enable 'Red Hat Enterprise Linux Server 7 RHSCL (RPMs)'
+   sudo yum -y  localinstall ~/foglamp-1.5.2-0.00.x86_64.rpm
+
+
+Installation on CentOS
+======================
+
+A CentOS package should be installed before the FogLAMP RPM, follow the instructions :
```
Comment (MarkRiddoch) [inline docs/REDHAT.rst]: It is necessary to install a CentOS package before FogLAMP can be installed successfully. The installation sequence is as follows:

File: README.rst (hunk 8 above)
Comment (sstefano69) [reply README.rst]: done

File: docs/REDHAT.rst (hunk 9 above)
Comment (sstefano69) [reply docs/REDHAT.rst]: done

File: docs/REDHAT.rst (hunk 10 above)
Comment (sstefano69) [reply docs/REDHAT.rst]: done

File: docs/REDHAT.rst (hunk 11 above)
Comment (sstefano69) [reply docs/REDHAT.rst]: done

**Pull request merged**